scripts/generate-release-notes.js

```diff
@@ -80,13 +84,14 @@ async function finalizeReleaseNotes (tag, app) {
 
   // Make sure that the tag passed in is valid
   let tags = await run('git', ['tag'])
-  if (!tags.includes(tag)) {
+
+  if (!tags.split('\n').includes(tag)) {
```

Comment:
This ensures that we do a full match on items in the array instead of whether a string includes another string. This would allow something like `Student-6.5` get through when the tags includes `Student-6.5.1`